$MirOS: ports/print/cups/patches/patch-filter_imagetops_c,v 1.1.7.1 2005/03/18 15:51:00 tg Exp $
--- filter/imagetops.c.orig	Fri Apr  7 13:48:14 2006
+++ filter/imagetops.c	Tue May  9 17:39:40 2006
@@ -547,7 +547,7 @@ main(int  argc,				/* I - Number of comm
     * Set the new custom size...
     */
 
-    sprintf(s, "Custom.%.0fx%.0f", width, length);
+    snprintf(s, sizeof(s), "Custom.%.0fx%.0f", width, length);
     ppdMarkOption(ppd, "PageSize", s);
 
    /*
